Error: prefix already exists: /home/ubuntu/miniconda/envs/_pipbuild_
Collecting SQLAlchemy-Utils==0.30.11
  Could not find a version that satisfies the requirement SQLAlchemy-Utils==0.30.11 (from versions: 0.1, 0.2, 0.3, 0.5, 0.6, 0.7, 0.7.2, 0.7.3, 0.7.4, 0.7.5, 0.7.6, 0.7.7, 0.8.0, 0.8.1, 0.8.2, 0.8.3, 0.8.4, 0.9, 0.9.1, 0.10.0, 0.11.0, 0.12.0, 0.12.1, 0.12.2, 0.12.3, 0.12.4, 0.12.5, 0.13.0, 0.13.1, 0.13.2, 0.13.3, 0.14.0, 0.14.1, 0.14.3, 0.14.4, 0.14.5, 0.14.6, 0.14.7, 0.15.0, 0.15.1, 0.16.0, 0.16.1, 0.16.2, 0.16.3, 0.16.4, 0.16.5, 0.16.6, 0.16.7, 0.16.8, 0.16.9, 0.16.10, 0.16.11, 0.16.12, 0.16.13, 0.16.14, 0.16.16, 0.16.17, 0.16.18, 0.16.19, 0.16.20, 0.16.21, 0.16.22, 0.16.23, 0.16.24, 0.16.25, 0.19.0, 0.20.0, 0.22.0, 0.22.1, 0.23.0, 0.23.1, 0.23.2, 0.23.3, 0.23.5, 0.24.0, 0.24.1, 0.24.2, 0.24.3, 0.24.4, 0.25.0, 0.25.1, 0.25.2, 0.25.3, 0.25.4, 0.26.0, 0.26.2, 0.26.3, 0.26.5, 0.26.6, 0.26.7, 0.26.8, 0.26.9, 0.26.10, 0.26.11, 0.26.12, 0.26.13, 0.26.14, 0.26.15, 0.26.16, 0.26.17, 0.27.0, 0.27.1, 0.27.2, 0.27.3, 0.27.4, 0.27.5, 0.27.6, 0.27.7, 0.27.8, 0.27.9, 0.27.10, 0.27.11, 0.28.0, 0.28.1, 0.28.2, 0.28.3, 0.29.0, 0.29.1, 0.29.2, 0.29.3, 0.29.4, 0.29.5, 0.29.6, 0.29.7, 0.29.8, 0.29.9, 0.30.0, 0.30.1)
No matching distribution found for SQLAlchemy-Utils==0.30.11
Package search: {'_pypi_ordering': False, 'version': '0.30.11', 'name': 'SQLAlchemy-Utils', 'summary': 'Various utility functions for SQLAlchemy.'}
Creating standard recipe for sqlalchemy-utils-0.30.11
Using url https://pypi.python.org/packages/source/S/SQLAlchemy-Utils/SQLAlchemy-Utils-0.30.11.tar.gz (105 KB) for SQLAlchemy-Utils.
Downloading SQLAlchemy-Utils
Unpacking SQLAlchemy-Utils...
done
working in /tmp/tmpczF0OQconda_skeleton_SQLAlchemy-Utils-0.30.11.tar.gz
Fetching package metadata: ......

WARNING: conda-build appears to be out of date. You have version 1.13.0 but the
latest version is 1.14.0. Run

conda update -n root conda-build

to get the latest version.

Solving package specifications: .patching file core.py
Hunk #1 succeeded at 168 with fuzz 2 (offset 2 lines).
Traceback (most recent call last):
  File "setup.py", line 7, in <module>
    from setuptools import setup, find_packages
ImportError: No module named setuptools

Applying patch: '/tmp/tmpczF0OQconda_skeleton_SQLAlchemy-Utils-0.30.11.tar.gz/pypi-distutils.patch'
$PYTHONPATH = /tmp/tmpczF0OQconda_skeleton_SQLAlchemy-Utils-0.30.11.tar.gz/SQLAlchemy-Utils-0.30.11
Error: command failed: /home/ubuntu/miniconda/envs/_build/bin/python setup.py install

conda create -n _pipbuild_ --yes python pip
/home/ubuntu/miniconda/envs/_pipbuild_/bin/pip install SQLAlchemy-Utils==0.30.11
Error: Could not pip install SQLAlchemy-Utils==0.30.11
